DorisProject/src/environments/environment.prod.ts

```diff
@@ -6,5 +6,6 @@ export const environment = {
   saveProduct: '/api/saveProduct',
   getProductSku: '/api/getProduct/',
   getCategoryList: '/api/getCategoryList/',
-  saveCategory: '/api/category'
+  saveCategory: '/api/category',
+  deleteCategory: '/api/deleteCategoryByIdJpa'
```

Comment:
Codacy has a fix for the issue: [Insert `,`](https://app.codacy.com/gh/SaraArenasV/DorisProjectFrontEnd/pullRequest?prid=9283869)
```suggestion
  deleteCategory: '/api/deleteCategoryByIdJpa',
```